manifests/cf-manifest/operations.d/730-elasticache-broker.yml

```diff
@@ -71,6 +71,12 @@
                       free: true
                       metadata:
                         displayName: Redis Non-clustered Tiny
+                    - id: b6949ea7-5c98-4c69-b981-4b5318ea8b7c
+                      name: medium-clustered-3.2
+                      description: 6.37GB RAM, clustered (1 shard), 3 nodes, failover, daily backups
+                      free: true
```

Comment:
I don't think we want this to be marked as free - this will make it available to trial orgs.